src/index.html

```diff
@@ -21,7 +24,322 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body page__body">
+    <header class="header">
+      <a
+        href="#Home"
+        class="header__img"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+      <nav class="header__nav">
+        <a
+          href="#Apple"
+          class="header__link is-active"
+        >
+          Apple
+        </a>
+        <a
+          href="#Samsung"
+          class="header__link"
+        >
+          Samsung
+        </a>
+        <a
+          href="#Smartphones"
+          class="header__link"
+        >
+          Smartphones
+        </a>
+        <a
+          href="#Laptops & Computers"
+          class="header__link"
+          data-qa="nav-hover"
+        >
+          Laptops & Computers
+        </a>
+        <a
+          href="#Gadgets"
+          class="header__link"
+        >
+          Gadgets
+        </a>
+        <a
+          href="#Tablets"
+          class="header__link"
+        >
+          Tablets
+        </a>
+        <a
+          href="#Photo"
+          class="header__link"
+        >
+          Photo
+        </a>
+        <a
+          href="#Video"
+          class="header__link"
+        >
+          Video
+        </a>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+        class="card catalog__card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="imac image"
+        />
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__review">
+          <div class="stars card__review-stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review-text">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-numbers">$2,199</div>
+        </div>
+        <a
+          class="card__buy"
+          href="#Buy"
+          data-qa="card-hover"
```

Comment:
The `data-qa` attribute is used here for the 'Buy' button. Ensure that this attribute is consistently applied across all similar elements for testing or automation purposes.